### PR TITLE
Mapwize SDK 2.0.0 and improved UI

### PR DESCRIPTION
**Pull request by emerciercontexeo**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,18 +1,28 @@
 # Mapwize UI iOS Changelog
 
-## Version 1.0.3
+## 1.1.0
+
+- Adding support for MapwizeForMapbox 2.0.0
+- Adding details in bottom view if the selected content have details to display
```
Comment (mathieugerard) [inline changelog.md]: has details

File: MapwizeUI/MapwizeView/MWZMapwizeView.h [hunk 2]
```diff
@@ -15,6 +15,16 @@ NS_ASSUME_NONNULL_BEGIN
     
 - (void) setIndoorLocationProvider:(ILIndoorLocationProvider*) indoorLocationProvider;
 
+- (void) grantAccess:(NSString*) accessKey success:(void (^)(void)) success failure:(void (^)(NSError* error)) failure;
+
+- (void) selectPlace:(MWZPlace*) place centerOn:(BOOL) centerOn;
```
Comment (mathieugerard) [inline MapwizeUI/MapwizeView/MWZMapwizeView.h]: Does it need to be in the doc?

File: MapwizeUI/MapwizeView/MWZMapwizeView.h [hunk 3]
```diff
@@ -15,6 +15,16 @@ NS_ASSUME_NONNULL_BEGIN
...
+
+- (void) selectPlaceList:(MWZPlaceList*) placeList;
```
Comment (mathieugerard) [inline MapwizeUI/MapwizeView/MWZMapwizeView.h]: Does it need to be in the doc?

File: MapwizeUI/MapwizeView/MWZMapwizeView.h [hunk 4]
```diff
@@ -15,6 +15,16 @@ NS_ASSUME_NONNULL_BEGIN
...
+
+- (void) unselectContent:(BOOL) closeInfo;
```
Comment (mathieugerard) [inline MapwizeUI/MapwizeView/MWZMapwizeView.h]: Does it need to be in the doc?

File: MapwizeUI/MapwizeView/MWZMapwizeView.h [hunk 5]
```diff
@@ -15,6 +15,16 @@ NS_ASSUME_NONNULL_BEGIN
...
+
+- (void) setDirection:(MWZDirection*) direction from:(id<MWZDirectionPoint>) from to:(id<MWZDirectionPoint>) to isAccessible:(BOOL) isAccessible;
```
Comment (mathieugerard) [inline MapwizeUI/MapwizeView/MWZMapwizeView.h]: Does it need to be in the doc?

File: MapwizeUI/DirectionModule/MWZComponentDirectionBarDelegate.h [hunk 6]
```diff
@@ -6,8 +6,10 @@
 @protocol MWZComponentDirectionBarDelegate <NSObject>
     
 - (void) didPressBack;
-- (void) didStartDirection:(MWZDirection*) direction from:(id<MWZDirectionPoint>) from to:(id<MWZDirectionPoint>) to;
+- (void) didUpdateDirectionInfo:(double) travelTime distance:(double) distance;
 - (void) didStopDirection;
+- (void) didStartLoading;
```
Comment (mathieugerard) [inline MapwizeUI/DirectionModule/MWZComponentDirectionBarDelegate.h]: Does it need to be in the doc?

File: MapwizeUI/MapwizeView/MWZMapwizeView.h [hunk 7]
```diff
@@ -15,6 +15,16 @@ NS_ASSUME_NONNULL_BEGIN
     
 - (void) setIndoorLocationProvider:(ILIndoorLocationProvider*) indoorLocationProvider;
 
+- (void) grantAccess:(NSString*) accessKey success:(void (^)(void)) success failure:(void (^)(NSError* error)) failure;
```
Comment (mathieugerard) [inline MapwizeUI/MapwizeView/MWZMapwizeView.h]: Does it need to be in the doc?

File: MapwizeUI/ResultList/MWZComponentGroupedResultList.m [hunk 8]
```diff
@@ -14,11 +15,13 @@ @implementation MWZComponentGroupedResultList {
     MWZUniverse* activeUniverse;
     NSMutableDictionary<NSString*, NSMutableArray<id<MWZObject>>*>* contentByUniverseId;
     NSLayoutConstraint* tableHeightConstraint;
+    NSString* language;
 }
 
 - (instancetype)init {
     self = [super init];
     if (self) {
+        language = @"en";
```
Comment (mathieugerard) [inline MapwizeUI/ResultList/MWZComponentGroupedResultList.m]: Why do we need a default?

File: Podfile.lock [hunk 9]
```diff
@@ -1,14 +1,16 @@
...
-  - MapwizeForMapbox (1.8.1):
+  - Mapbox-iOS-SDK (4.7.1)
+  - MapwizeForMapbox (2.0.0):
     - IndoorLocation (= 1.0.4)
-    - Mapbox-iOS-SDK (= 4.6.0)
-  - MapwizeUI (1.0.2):
-    - MapwizeForMapbox (~> 1.8)
+    - Mapbox-iOS-SDK (= 4.7.1)
+    - SSZipArchive (= 2.1.4)
+  - MapwizeUI (1.0.3):
```
Comment (mathieugerard) [inline Podfile.lock]: Nobody cares about the podfile.lock but ...

File: MapwizeUI/DirectionModule/MWZComponentDirectionBarDelegate.h (hunk 6 above)
Comment (emerciercontexeo) [reply MapwizeUI/DirectionModule/MWZComponentDirectionBarDelegate.h]: private use only

**Pull request merged**